Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "ParametricOptInterface"
 uuid = "0ce4ce61-57bf-432b-a095-efac525d185e"
 authors = ["Tomás Gutierrez <tomasfmgutierrez@gmail.com>"]
-version = "0.9.0"
+version = "0.10.0"
```

Comment:
If this is a new feature, it should be `0.9.1`. If this is breaking, it should be `0.10.0`

Reply:
Honestly, you could also see it as either one or the other. I tended to the breaking change bump because before the behavior was an error and now it is allowed. It is not something that did not exist in the package and now it is implemented. It was explicitly prohibited.